dists/amigaos/md2ag.rexx

```diff
@@ -0,0 +1,136 @@
+/*
+$VER: md2ag.rexx 0.1 (23.05.2021) README(.md) to (amiga).guide converter.
+Converts a given markdown README file to a basic hypertext Amiga guide file
+and installs it to a given location.
+*/
+
+PARSE ARG p_readme p_instpath
+
+/*
+Check if arguments are available, otherwise quit.
+*/
+IF ~ARG() THEN DO
+	SAY 'No Arguments given!'
+	SAY 'Usage: md2ag.rexx README.md INSTALLPATH'
+	EXIT
+END
+
+/*
+If the given filename/path has spaces in it, AmigaDOS/CLI
+will add extra quotation marks to secure a sane working path.
+Get rid of them to make AREXX find the file and remove leading
+and trailing spaces.
+*/
+IF ~EXISTS(p_readme) THEN DO
+	SAY p_readme' not available!'
+	EXIT
+END
+ELSE DO
+	p_readme=STRIP(p_readme)
+	p_readme=COMPRESS(p_readme,'"')
+END
+IF p_instpath='' THEN DO
+	SAY 'No installation destination given!'
+	EXIT
+END
+ELSE DO
+	p_instpath=STRIP(p_instpath)
+	p_instpath=STRIP(p_instpath,'T','/')
+	p_instpath=COMPRESS(p_instpath,'"')
+END
+
+CALL OPEN read_md,p_readme,'R'
+
+IF READCH(read_md,18)~='# [ScummVM README]' THEN DO
+	CALL CLOSE read_md
+	SAY p_readme' is not the ScummVM README.md file. Aborting!'
+	EXIT
+END
+ELSE
+	/*
+	Skip the first "Build Status" line.
+	*/
+	CALL READLN read_md
+
+CALL OPEN write_guide,'README.guide','W'
+
+/*
+Prepare Amiga guide, add intro and fixed text.
+*/
+CALL WRITELN write_guide,'@DATABASE ScummVM README.guide'
+CALL WRITELN write_guide,'@$VER: ScummVM Readme 2.3.0git'
+CALL WRITELN write_guide,'@(C) by The ScummVM team'
+CALL WRITELN write_guide,'@AUTHOR The ScummVM team'
+CALL WRITELN write_guide,'@WORDWRAP'
+CALL WRITELN write_guide,'@NODE "main" "ScummVM README Guide"'
+CALL WRITELN write_guide,'@{b}'
+CALL WRITELN write_guide,'ScummVM README'
+CALL WRITELN write_guide,'@{ub}'
+
+DO WHILE ~EOF(read_md)
+	v_line=READLN(read_md)
+
+	/*
+	Check the one "rolled over" line and rejoin it before progressing.
+	It's lines 11, 12 and 13 in the opriginal .md, but we only join 12 and 13
+	due to the cut weblink:
+	latest release, progress reports and more, please visit the ScummVM [home
+	page](https://www.scummvm.org/).
+	*/
+	IF POS('[',v_line)>0 THEN DO
+		IF POS(']',v_line)=0 THEN DO
+			rejoin_line=READLN(read_md)
+			v_line=v_line''rejoin_line
+		END
+	END
+
+	/*
+	Replace any  '�' characters with '`'.
```

Comment:
This character was unfortunately changed during copying (or utf did something to it).
On my system it looks like an "a" with "^" on top, but it seems it gets converted somehow.

Reply:
Huuuum, I wondered what it was.
It's mangled because Github editor expects UTF-8 but AmigaOS is still on old single-byte encodings.
Do you know which byte code it is?

Reply:
OK found it. It's the UTF-8 sequence for ’ character. It's a 3 bytes UTF-8 sequence e2 80 99. And you would like to translate it to a 1 byte.
It seems OVERLAY doesn't do this. Do you have an idea?
Else, we could patch the README to make it use ASCII only characters. We must see with the team for this though.